mongodb_statedb/__init__.py

```diff
@@ -82,6 +100,41 @@ def delete_all(self) -> None:
         """Delete all of the key-value pairs stored in the used collection."""
         self._db.drop()
 
+    def clear(self, key: str) -> None:
```

Comment:
I'm not sure that there is a good reason to have these clear methods. Is there a case where you want empty values in the database? If so, see my above comment about adding an `isSet` field to the mongodb entry, as that seems like the propper way to do this to me.

A lot of times there are things that seem nice to have, but aren't really useful to our usecase. Do you think this is important in how you plan to design the interaction?

Reply:
The biggest use for `clear` is to reset the state database for the demo interactions, as we don't want to save the previous user's name for the next interaction.